02-IDE_exercises/solutions/07b-project/reverse_list.py

```diff
@@ -2,13 +2,20 @@
 # Date: 10/29/2020
 # Description: Mutates a given list and by the order of the elements of teh list
 
-def reverse_list(lst):
+def reverse_list(lst: object) -> None:
```

Comment:
this is type hinting a parameter named `lst` of any type. I think you mean `lst: list[object]`